frontend/src/components/application/Details.stories.tsx

```diff
@@ -2,15 +2,14 @@ import React from "react"
 import { ComponentMeta } from "@storybook/react"
 import Details from "./Details"
 import { faker } from "@faker-js/faker"
-import { AppStats } from "src/types/AppStats"
```

Comment:
Why these changes? Maybe revert them?

Reply:
I made them as a consequence of having the "Cannot find module" error for AppStats, Appstream and Summary. I can change them as they were before, but the error would appear again

Reply:
Where did you get that error? I don't think I can replicate and CI was fine before too.

Reply:
NVM, figured it out. Ideally this should be in a different MR.